backend/src/openbeheer/zaaktype/tests/test_zaaktype_detail.py

```diff
@@ -0,0 +1,512 @@
+            )
+            self.assertEqual(zaaktype["besluittypen"], [])
+            self.assertEqual(zaaktype["gerelateerdeZaaktypen"], [])
+            self.assertEqual(
+                zaaktype["url"],
+                "http://localhost:8003/catalogi/api/v1/zaaktypen/ce9feadd-00cb-46c8-a0ef-1d1dfc78586a",
+            )
+            self.assertEqual(zaaktype["identificatie"], "ZAAKTYPE-2018-0000000002")
+            self.assertEqual(zaaktype["omschrijvingGeneriek"], "")
+            self.assertEqual(zaaktype["toelichting"], "")
+            self.assertIsNone(zaaktype["servicenorm"])
+            self.assertEqual(zaaktype["verlengingstermijn"], "P40D")
+            self.assertEqual(zaaktype["trefwoorden"], [])
+            self.assertEqual(zaaktype["publicatietekst"], "")
+            self.assertEqual(zaaktype["verantwoordingsrelatie"], [])
+            self.assertEqual(
+                zaaktype["selectielijstProcestype"],
+                "https://selectielijst.openzaak.nl/api/v1/procestypen/c844637e-6393-4202-b030-e1bffb08a9b0",
+            )
+            self.assertIsNone(zaaktype["broncatalogus"])
+            self.assertIsNone(zaaktype["bronzaaktype"])
+            self.assertIsNone(zaaktype["eindeGeldigheid"])
+            self.assertEqual(zaaktype["beginObject"], "2018-01-01")
+            self.assertIsNone(zaaktype["eindeObject"])
+            self.assertEqual(
+                zaaktype["statustypen"],
+                [
+                    "http://localhost:8003/catalogi/api/v1/statustypen/835a2a13-f52f-4339-83e5-b7250e5ad016"
+                ],
+            )
+            self.assertEqual(
+                zaaktype["resultaattypen"],
+                [
+                    "http://localhost:8003/catalogi/api/v1/resultaattypen/5d39b8ac-437a-475c-9a76-0f6ae1540d0e"
+                ],
+            )
+            self.assertEqual(zaaktype["eigenschappen"], [])
+            self.assertEqual(
+                zaaktype["informatieobjecttypen"],
+                [
+                    "http://localhost:8003/catalogi/api/v1/informatieobjecttypen/9dee6712-122e-464a-99a3-c16692de5485"
+                ],
+            )
+            self.assertEqual(zaaktype["roltypen"], [])
+            self.assertEqual(zaaktype["deelzaaktypen"], [])
+            self.assertEqual(zaaktype["zaakobjecttypen"], [])
+
+    def test_proxy_error_response(self):
+        endpoint = reverse(
+            "api:zaaktype-detail",
+            kwargs={"slug": "OZ", "uuid": "ce9feadd-00cb-46c8-a0ef-1d1dfc78586a"},
+        )
+        self.client.force_login(self.user)
+
+        response = self.client.put(
+            endpoint,
+            data={
+                "omschrijving": "MODIFIED",
+                # Missing all the other required fields.
+            },
+            format="json",
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
400? What did the client do wrong here? What's wrong with the request?

Reply:
For the PUT, you need to give all of the required fields, while here only `omschrijving` is passed (at the end of the test, you can see the loop over all the error to check for the error code `required`).